src/base/io/log.cpp

```diff
@@ -1,12 +1,35 @@
 #include "log.h"
 
+#include <ctime>
+
 using namespace IO;
 
-Log::LogFunc Log::logFunc;
+namespace
+{
+static bool enabled = true;
+static Log::LogFunc logFunc = [](std::string const &s)
+{
+	puts(s.c_str());
+};
+
+}
+
+LogRecord::LogRecord() : content("[vizzu] [YYYY-mm-ddTHH:MM:SSZ] ")
```

Comment:
I'm not sure if we still need the timestamp in the log message. I don't see timestamps usually in the console log.

Reply:
I added a new flag 'setTimestamp'